frontend/src/widgets/Calendar/ReviewTimeTable.vue

```diff
@@ -23,7 +23,11 @@ export default {
     methods: {
     	start: function() 
 	{
-		window.location.href = "/frontend/#/review";
+    localStorage.setItem('careIdFromCalendar', this.care.id);
+    localStorage.setItem('patientIdFromCalendar', this.care.patient.id);
```

Comment:
ne ovako nego imas view koji prima parametar (props: ["careId", "patientId"]) i onda redirektujes sa this.$router.push("/createCare/" + patientId + "/" + careId) a u tom view-u kazes da prima ta dva propsa i u router.js kazes recimo {path: "/createCare/:patientId/:careId"} i radice redirekt lepo